test/E2ETests/Common/DeploymentUtility.cs

```diff
@@ -201,7 +201,7 @@ private static Process StartMonoHost(StartParameters startParameters, ILogger lo
 
             var hostProcess = Process.Start(startInfo);
             logger.WriteInformation("Started {0}. Process Id : {1}", hostProcess.MainModule.FileName, hostProcess.Id);
-            Thread.Sleep(15 * 1000);
+            Thread.Sleep(25 * 1000);
```

Comment:
On windows we wait for the local db file to be created. On mono since we use inmemory store we can't apply the same logic. For now looks good to me :shipit: